qiita_ware/ebi.py

```diff
@@ -674,25 +644,7 @@ def send_xml(self):
                 study_accession = accessions.group('study')
                 submission_accession = accessions.group('submission')
 
-                LogEntry.create('Runtime', "Study accession:\t%s" %
-                                study_accession)
-                LogEntry.create('Runtime', "Submission accession:\t%s" %
-                                submission_accession)
-
-                print "Study accession:\t", study_accession
-                print "Submission accession:\t", submission_accession
-            else:
-                LogEntry.create('Runtime', ("However, the accession numbers "
-                                            "could not be found in the output "
-                                            "above."))
-                print ("However, the accession numbers could not be found in "
-                       "the output above.")
-        else:
-            LogEntry.create('Fatal', curl_result)
-            print curl_result
-            print "FAILED"
-
-        return (study_accession, submission_accession)
+        return study_accession, submission_accession
```

Comment:
Sounds good, but then we need to make sure that the actual `curl_result` is also stored in the error